components/motor/gpio/motor_encoder.go

```diff
@@ -482,6 +485,7 @@ func (m *EncodedMotor) computeRamp(oldPower, newPower float64) float64 {
 // Both the RPM and the revolutions can be assigned negative values to move in a backwards direction.
 // Note: if both are negative the motor will spin in the forward direction.
 func (m *EncodedMotor) GoFor(ctx context.Context, rpm, revolutions float64, extra map[string]interface{}) error {
+	m.opMgr.New(ctx)
```

Comment:
is this call conflicting with the op created below (line 497?) Plus aren't you ignoring the retruns?

Reply:
good catch, this should not be here.